docs/developer_guide.md

```diff
@@ -41,17 +46,17 @@ make deploy IMG=registry.toolbox.iotg.sclab.intel.com/cpp/operator:latest
 ## OLM development flow
 ```
 make cluster_clean
-make build_all_images
+make build_all_images DROP_NOTEBOOK=0
```

Comment:
I think it's ADD_NOTEBOOK_K8S now and it's 0 by default so maybe we can drop this?